carbon/app.py

```diff
@@ -244,6 +244,7 @@ def run_connection_test(self) -> None:
                 f"Failed to connect to the Symplectic Elements FTP server: {error}"
             )
             logger.exception(error_message)
+            raise
```

Comment:
+1 - think it's good that application bails given a connection error, as it cannot really do its work without it